src/clarity-icons/helpers.spec.ts

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright (c) 2016-2017 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { ClarityIcons } from "./index";
+import { CoreShapes } from "./shapes/core-shapes";
+import * as DomPurify from "dompurify";
+
+
+// TODO: open question... should we make whitespace removal part of the icon parsing???
+export function removeWhitespace(htmlStr: string): string {
+    let returnStr = htmlStr.replace(/[\n|\r]+/g, "");
+    returnStr = returnStr.replace(/>[ |\t]+</g, "><");
+    returnStr = returnStr.replace(/"[ |\t]+/g, "\" ");
+    return returnStr.trim();
+}
+
+const sanitizeOptions = {
```

Comment:
I forgot to say this on the last review. But thanks for breaking this out. I appreciate it!